src/router/router.ts

```diff
@@ -102,11 +102,11 @@ export function createRouter(baseGuiPath: string = '/', policyDefinitions: Polic
               name: 'gateway-list-view',
               meta: {
                 title: 'Gateways',
-                type: 'gateway',
               },
               props: (route) => ({
                 name: route.query.name,
                 offset: getLastNumberParameter(route.query.offset),
+                isGatewayView: true,
```

Comment:
As mentioned offline, much prefer using `props` for this, weirdly I couldn't seem to get it working with props on my first attempt, I must have been doing something wrong!